### ISSUE TITLE
Fail updates with "Cannot read property 'latest' of undefined"

### ISSUE THREAD

**Issue by melpon** [silverwind/updates#7]
If `package.json` includes `@reach/router` registry, fail `npx updates`.

```
$ cat package.json
(snip)
  "dependencies": {
    "@reach/router": "^1.1.1"
  }
(snip)
$ npx updates
Cannot read property 'latest' of undefined
```

It seems to be because the [@reach/router registry](https://registry.npmjs.org/@reach/router) returns 404 and [fetch response status](https://github.com/silverwind/updates/blob/v4.1.0/updates.js#L127) is not checked it is 200.

I don't know why @reach/router registry returns 404.

**Comment by melpon** [comment]
I found that I have to convert a dep name `/` to `%2f` like [this](https://github.com/npm/cli/blob/v6.2.0/lib/utils/map-to-registry.js#L17).

**Comment by silverwind** [comment]
Thanks, fixed in `4.1.1`.
